packages/taquito/test/contract/rpc-estimate-provider.spec.ts

```diff
@@ -96,7 +97,7 @@ describe('RPCEstimateProvider test', () => {
   });
 
   describe('originate', () => {
-    it('should produce a reveal and origination operation', async done => {
+    it('should produce an origination operation, no reveal', async done => {
```

Comment:
By default, reveal is done on operations. Why is it that here origination operation has no reveal. Also this test doesn't test for if there is any reveal (Line 122).

Reply:
Reveal is done only if the account needs a reveal. This test simulates that the account does not need a reveal, so there is no reveal operation added. There is a case with an origination where a reveal is needed at line 241 below.